operations/overlay_compute_ngd.yml

```diff
@@ -0,0 +1,36 @@
+id: overlay_compute_ngd
+subclass_of: overlay
+name: overlay compute ngd
+description: This operation computes the normalized Google distance (co-occurrence frequency) in PubMed abstracts and adds virual edges between qnodes AND/OR knodes AND/OR results edge bindings.
+examples: 
+- input: overlay/messages/07_input_ngd.json
+  output: overlay/messages/08_output_ngd.json
+output_guarantees:
+- At least 1 new edge between qnodes
```

Comment:
None of our existing guarantees is described relative to the input like this one is. It's going to be very difficult to validate. Is this guarantee necessary or useful to the client? What happens if a client asks to apply overlay_compute_ngd twice in succession? Will it end up with a bunch of duplicate edges?

Reply:
That's a good point. I can remove this guarantee as well.

Reply:
I had overlooked the desire for idempotency of this operation. I agree that this output guarantee should be removed.

Reply:
This has now been removed.